### ISSUE TITLE
Theme Upload with no arguments should upload entire theme to Shopify

### ISSUE THREAD

**Issue by bman-80stees** [csaunders/themekit#48]
I was trying to upload the entire theme back to Shopify and nothing happened when I tried these:
```theme upload```
```theme upload -env="development"```
downloading the theme works just fine and uploading individual files works as well.  Just the uploading all files does not work. Is this supposed to work or is it a bug?

**Comment by csaunders** [comment]
It's not supported at the moment.

**Comment by bman-80stees** [comment]
I was asking around and everyone that I spoke with said it was based off of the theme gem and `theme upload` for all files works with that.  The reason I asked was because, I had over 25 files to upload and didn't want to do them individually. That would take way too much time and trying to remember which ones I did and all that. I also didn't want to use the `theme watch` as I needed to make multiple changes and didn't want that to get me into trouble uploading something I didn't intend.

**Comment by csaunders** [comment]
Fixed in [0.2.3](https://github.com/csaunders/themekit/releases/tag/v0.2.3)

**Comment by thisislawatts** [comment]
Has there been a regression on this, just attempted the `theme`

```bash
theme upload

----------------------------------------------------------------
[ThemeKit - Shopify Theme Utilities v0.2.6 Author: Chris Saunders]
----------------------------------------------------------------

2015/07/06 08:52:13 invalid argument
```
